platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/style/BackgroundLayerTest.java

```diff
@@ -4,6 +4,7 @@
 
 import android.graphics.Color;
 import android.support.test.espresso.Espresso;
```

Comment:
many of them are removed but not all, some generated tests uses classes that aren't used by others.